clusters/kepler.yaml

```
@@ -0,0 +1,37 @@
+# Refer: https://artifacthub.io/packages/helm/kepler/kepler?modal=install
+# Note: By default kube-prometheus does not let you scrape services deployed in namespaces other than monitoring. 
+# So if you are running Kepler outside monitoring update the kube-prometheus
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: kepler
+  namespace: flux-system
+spec:
+  interval: 24h
+  url: https://sustainable-computing-io.github.io/kepler-helm-chart
+---
+# https://fluxcd.io/flux/components/helm/helmreleases/#install-configuration
+apiVersion: helm.toolkit.fluxcd.io/v2beta2
```

```suggestion
apiVersion: helm.toolkit.fluxcd.io/v2beta1
```

Cluster is running Flux v2.1.2 so we need to use v2beta1 until we upgrade.